--- packagechooser.conf.old	2025-03-17 00:00:47.000000000 +0100
+++ packagechooser.conf.new	2025-10-22 20:45:05.935703978 +0200
@@ -145,6 +145,7 @@
             - plasma-pa
             - plasma-systemmonitor
             - plasma-workspace
+            - plasma-x11-session
             - powerdevil
             - print-manager
             - qt6-imageformats
@@ -213,7 +214,6 @@
             - gnome-keyring
             - gnome-nettool
             - gnome-power-manager
-            - gnome-screenshot
             - gnome-shell
             - gnome-system-monitor
             - gnome-terminal
@@ -230,7 +230,7 @@
             - gvfs-smb
             - nautilus
             - sushi
-            - totem
+            - showtime
             - xdg-desktop-portal-gnome
             - xdg-desktop-portal
             - xdg-user-dirs-gtk
@@ -651,20 +651,20 @@
           - gvfs-nfs
           - gvfs-smb
           - lxappearance-gtk3
-          - obconf
+          - lxappearance-obconf-gtk3
           - lxde-common
           - lxde-icon-theme
-          - lxhotkey-gtk3
+          - lxhotkey
           - lxinput-gtk3
-          - lxlauncher-gtk3
-          - lxpanel-gtk3
+          - lxlauncher
+          - lxpanel
           - lxrandr-gtk3
-          - lxsession-gtk3
-          - lxtask-gtk3
+          - lxsession
+          - lxtask
           - lxterminal
           - notification-daemon
           - openbox
-          - pcmanfm-gtk3
+          - pcmanfm
           - lxmusic
           - mpv
           - network-manager-applet
@@ -715,6 +715,7 @@
             - arc-gtk-theme-eos
             - archlinux-xdg-menu
             - awesome-terminal-fonts
+            - brightnessctl
             - dex
             - dmenu
             - dunst
@@ -756,7 +757,6 @@
             - xdg-user-dirs-gtk
             - xed
             - xfce4-terminal
-            - xorg-xbacklight
             - xorg-xdpyinfo
             - xss-lock
             - zip
