--- packagechooser.conf.old	2025-03-17 00:00:47.000000000 +0100
+++ packagechooser.conf.new	2025-11-07 22:13:53.599315522 +0100
@@ -134,8 +134,7 @@
             - kscreen
             - kwallet-pam
             - kwayland-integration
-            - libappindicator-gtk3
-            - maliit-keyboard
+            - libappindicator
             - okular
             - plasma-browser-integration
             - plasma-desktop
@@ -145,6 +144,7 @@
             - plasma-pa
             - plasma-systemmonitor
             - plasma-workspace
+            - plasma-x11-session
             - powerdevil
             - print-manager
             - qt6-imageformats
@@ -213,7 +213,6 @@
             - gnome-keyring
             - gnome-nettool
             - gnome-power-manager
-            - gnome-screenshot
             - gnome-shell
             - gnome-system-monitor
             - gnome-terminal
@@ -230,7 +229,7 @@
             - gvfs-smb
             - nautilus
             - sushi
-            - totem
+            - showtime
             - xdg-desktop-portal-gnome
             - xdg-desktop-portal
             - xdg-user-dirs-gtk
@@ -650,21 +649,21 @@
           - gvfs-mtp
           - gvfs-nfs
           - gvfs-smb
-          - lxappearance-gtk3
-          - obconf
+          - lxappearance
+          - lxappearance-obconf
           - lxde-common
           - lxde-icon-theme
-          - lxhotkey-gtk3
-          - lxinput-gtk3
-          - lxlauncher-gtk3
-          - lxpanel-gtk3
-          - lxrandr-gtk3
-          - lxsession-gtk3
-          - lxtask-gtk3
+          - lxhotkey
+          - lxinput
+          - lxlauncher
+          - lxpanel
+          - lxrandr
+          - lxsession
+          - lxtask
           - lxterminal
           - notification-daemon
           - openbox
-          - pcmanfm-gtk3
+          - pcmanfm
           - lxmusic
           - mpv
           - network-manager-applet
@@ -715,6 +714,7 @@
             - arc-gtk-theme-eos
             - archlinux-xdg-menu
             - awesome-terminal-fonts
+            - brightnessctl
             - dex
             - dmenu
             - dunst
@@ -756,7 +756,6 @@
             - xdg-user-dirs-gtk
             - xed
             - xfce4-terminal
-            - xorg-xbacklight
             - xorg-xdpyinfo
             - xss-lock
             - zip
