--- packagechooser.conf	2025-03-17 00:00:47.000000000 +0100
+++ packagechooser.conf.patched	2025-08-11 18:01:02.987338191 +0200
@@ -145,6 +145,7 @@
             - plasma-pa
             - plasma-systemmonitor
             - plasma-workspace
+            - plasma-x11-session
             - powerdevil
             - print-manager
             - qt6-imageformats
@@ -651,7 +652,7 @@
           - gvfs-nfs
           - gvfs-smb
           - lxappearance-gtk3
-          - obconf
+          - lxappearance-obconf-gtk3
           - lxde-common
           - lxde-icon-theme
           - lxhotkey-gtk3
@@ -715,6 +716,7 @@
             - arc-gtk-theme-eos
             - archlinux-xdg-menu
             - awesome-terminal-fonts
+            - brightnessctl
             - dex
             - dmenu
             - dunst
@@ -756,7 +758,6 @@
             - xdg-user-dirs-gtk
             - xed
             - xfce4-terminal
-            - xorg-xbacklight
             - xorg-xdpyinfo
             - xss-lock
             - zip
