--- packagechooser.conf	2024-03-25 16:16:10.849685485 +0100
+++ packagechooser.conf.patched	2024-03-25 16:17:26.613905573 +0100
@@ -110,33 +110,46 @@
             - breeze-gtk
             - dolphin
             - dolphin-plugins
+            - ffmpegthumbs
+            - fwupd
             - gwenview
             - haruna
-            - kcalc
             - kate
+            - kcalc
+            - kde-cli-tools
+            - kde-gtk-config
             - kdeconnect
+            - kdegraphics-thumbnailers
+            - kdenetwork-filesharing
             - kdeplasma-addons
-            - kde-gtk-config
             - kgamma
-            - khotkeys
-            - kimageformats5
+            - kimageformats
             - kinfocenter
-            - kinit
+            - kio-admin
+            - kio-extras
             - kio-fuse
             - konsole
             - kscreen
             - kwallet-pam
+            - kwayland-integration
+            - libappindicator-gtk3
+            - maliit-keyboard
             - okular
+            - plasma-browser-integration
             - plasma-desktop
             - plasma-disks
+            - plasma-firewall
             - plasma-nm
             - plasma-pa
             - plasma-systemmonitor
+            - plasma-workspace
             - powerdevil
             - print-manager
             - sddm-kcm
             - spectacle
+            - xdg-desktop-portal-kde
             - xsettingsd
+            - xwaylandvideobridge
 
     - id: GNOME-Desktop
       name: "GNOME"
@@ -216,6 +229,7 @@
             - nautilus-sendto
             - sushi
             - totem
+            - tracker3-miners
             - xdg-desktop-portal-gnome
             - xdg-desktop-portal
             - xdg-user-dirs-gtk
@@ -745,4 +759,3 @@
             - xorg-xbacklight
             - xorg-xdpyinfo
             - zip
-
