--- packagechooser.conf	2024-04-17 15:32:00.000000000 +0200
+++ packagechooser.conf_gemini_1.patched	2024-06-19 11:06:41.073989848 +0200
@@ -227,7 +227,6 @@
             - gvfs-nfs
             - gvfs-smb
             - nautilus
-            - nautilus-sendto
             - sushi
             - totem
             - xdg-desktop-portal-gnome
@@ -586,7 +585,6 @@
             - oxygen-icons
             - pavucontrol-qt
             - print-manager
-            - qt5-translations
             - vlc
             - xdg-utils
             - xscreensaver
@@ -737,7 +735,7 @@
             - jq
             - lightdm
             - lightdm-slick-greeter
-            - lxappearance-gtk3
+            - nwg-look
             - mpv
             - network-manager-applet
             - numlockx
